tortoise/queryset.py

```diff
@@ -658,7 +660,12 @@ def _make_query(self) -> None:
                     db_field = self.model._meta.fields_db_projection[key]
                 except KeyError:
                     raise FieldError(f"Field {key} is virtual and can not be updated")
-                value = executor.column_map[key](value, None)
+                if not isinstance(value, (F, ArithmeticExpression)):
+                    value = executor.column_map[key](value, None)
```

Comment:
Ok, so after investigating, I think due to F and ArithmeticExpression baving different interfaces. Both inherit `pypika.Term`, so maybe just check against that?